lms/templates/emails/activation_email.html

```diff
@@ -0,0 +1,25 @@
+<%inherit file="main.html" />
+<%! from django.utils.translation import ugettext as _ %>
+<%! from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers %>
+
+<%block name="message">
```

Comment:
Are you sure that we need to use this block?

Reply:
It's required block